api/controllers/game/concede.js

```diff
@@ -1,30 +1,28 @@
-module.exports = function (req, res) {
-  return gameService
-    .populateGame({ gameId: req.session.game })
-    .then(function clearGame(game) {
-      return Promise.all([
-        Promise.resolve(game),
-        gameService.clearGame({ userId: req.session.usr }),
-      ]);
-    })
-    .then(function publishAndRespond(values) {
-      const game = values[0];
-      var victory = {
-        gameOver: true,
-        winner: (req.session.pNum + 1) % 2,
-        conceded: true,
-      };
-      Game.publish([game.id], {
-        verb: 'updated',
-        data: {
-          change: 'concede',
-          game,
-          victory,
-        },
-      });
-      return res.ok();
-    })
-    .catch(function failed(err) {
-      return res.badRequest(err);
+module.exports = async function (req, res) {
+  try {
+    const winner = (req.session.pNum + 1) % 2;
```

Comment:
I wonder if we need a helper to manage pNum eventually. There's a lot of manual math to get the pnum and forgetting to add one and/or modulus it seems like an area that may be error prone.

Reply:
Open to it. Not sure what it'd look like. A helper to return your opponent's pNum maybe?